acrg/name/name.py

```diff
@@ -1562,30 +1563,34 @@ def local_solar_time(dataset):
         hours = dataset.time.to_pandas().index.hour
         return hours
     
-    def local_ratio(dataset):
```

Comment:
Just to check - is there no longer a function called `local_ratio` in the new version? Just wondering if we may want to just include this so the interface hasn't changed even if the details have - this could just be a very thin wrapper using the updates for instance.